### ISSUE TITLE
Feature request: qbec.io/update-policy annotation

### ISSUE THREAD

**Issue by kvaps** [splunk/qbec#69]
Hi, I was just thinking about how to secure from removal of important resources, eg `PersistentVolumeClaims`. In my case I want to be able to create this kind resources but do not remove them under what circumstances.

My idea is to specify special annotation for these kind resources:

```diff
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   annotations:
     qbec.io/component: pvc
+    qbec.io/update-policy: DoNotRemove
   labels:
     qbec.io/application: myapp
     qbec.io/environment: devel
   name: myclaim
 spec:
   accessModes:
   - ReadWriteOnce
   resources:
     requests:
       storage: 8Gi
   storageClassName: myclass
   volumeMode: Filesystem
```

I think we can support few more policies:
- `Normal` - Standard policy, used by default if annotation is not set
- `DoNotUpdate` - It might be useful for initial Jobs
- `DoNotRemove` - It might be useful for PVCs

On the client we can show warning like:
```
[warn]: delete persistentvolumeclaim myclaim (source pvc): skiping due DoNotRemove policy set
```
and add `skipped` to final stats:

```
stats:
  skipped:
  - persistentvolumeclaim myclaim (source pvc)
```

And add `--force` flag to handle these resources too

**Comment by kvaps** [comment]
I was just thinking little more about it, sometimes it might be useful to have both annotations set, what do you think about using:
```diff
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   annotations:
     qbec.io/component: pvc
+    qbec.io/do-not-remove: "true"
+    qbec.io/do-not-update: "true"
   labels:
     qbec.io/application: myapp
     qbec.io/environment: devel
   name: myclaim
 spec:
   accessModes:
   - ReadWriteOnce
   resources:
     requests:
       storage: 8Gi
   storageClassName: myclass
   volumeMode: Filesystem
```
instead of `qbec.io/update-policy`?

**Comment by gotwarlost** [comment]
Implemented in [v0.10](https://github.com/splunk/qbec/releases/tag/v0.10.0) 

See [user guide page](https://qbec.io/userguide/usage/directives/) for usage
